src/main/java/com/google/api/gax/core/BackoffParams.java

```diff
@@ -54,23 +56,23 @@ public Builder toBuilder() {
 
   @AutoValue.Builder
   public abstract static class Builder {
-    public abstract Builder setInitialDelayMillis(long initialDelayMillis);
+    public abstract Builder setInitialDelayDuration(Duration initialDelayDuration);
```

Comment:
Remove the 'Duration' from this method name (and for max delay too), it should be understood.